docs/_articles/en/apex/apex-debugging.md

```diff
@@ -5,12 +5,17 @@ lang: en
 
 The Salesforce Extension pack comes with rich debugging options for every developer, regardless of where you land on the low- to pro-code continuum. Use this information to figure out which Apex debugger is right for you:
 
-- [Replay Debugger](./en/apex/replay-debugger): Free and easy to use.
-- [Interactive Debugger](./en/apex/interactive-debugger): Traditional, licensed debugger that lets you view and pause events in real time.
-- [ISV Debugger](./en/apex/isv-debugger): Use to debug managed package code directly in customer orgs from your License Management Org (LMO).
+
+| Debugger              | License Requirements                                                                                                                                                                                                 | Features                                                                                                                      |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|
+| [Replay Debugger](./en/apex/replay-debugger)       | None. Free.                                                                                                                                                                                                          | Easy to use. Use debug logs to “replay” events. Your debugger for most use cases.                                             |
+| [Interactive Debugger](./en/apex/interactive-debugger)  | - One free license in Performance and Unlimited Edition orgs.<br>- Available for purchase in Enterprise Edition orgs.<br>- Not available in Trial and Developer Edition orgs.<br>- Can be used in scratch orgs using the DebugApex feature. To use the debugger in a scratch org, the associated dev hub org must have the interactive debugger license. | A powerful debugger that lets you view and pause events in real time.                                                         |
```

Comment:
```suggestion
| [Interactive Debugger](./en/apex/interactive-debugger)  | - One free license in Performance and Unlimited Edition orgs.<br>- Available for purchase in Enterprise Edition orgs.<br>- Not available in Trial and Developer Edition orgs.<br>- Can be used in scratch orgs using the DebugApex feature. To use the debugger in a scratch org, the associated Dev Hub org must have the interactive debugger license. | A powerful debugger that lets you view and pause events in real time.                                                         |
```

Reply:
Also, [this page](https://developer.salesforce.com/tools/vscode/en/apex/interactive-debugger) calls the license "Apex Debugger License" (right?). I find Salesforce licenses SO confusing, so if both pages are talking about the same thing, I suggest you use the same terminology.  And maybe use the same link to https://help.salesforce.com/s/articleView?id=000382135&type=1.

Reply:
Great catch! Let me confirm what the correct name for the license is and I'll update the doc.